api_mongoDB.go

```diff
@@ -10,8 +10,10 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
+	"fmt"
```

Comment:
Can you add Intel Copyright in appropriate files ?

Reply:
done